sympy/sets/sets.py

```diff
@@ -153,6 +153,181 @@ def isdisjoint(self, other):
         """
         return self.is_disjoint(other)
 
+    def _union_simplify(self, other):
+        """
+        Try to reduce number of imageset in the args. Helper method for
```

Comment:
Can you add the definition of this function stating the abstraction it provides ?